### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by SooDragon**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @SooDragon. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/12/11 17:04:06 Loaded 7562 libraries from DB
Changing URL of library Geekble_Note2Freq from https://github.com/geekblemaker/Geekble_note2freq.git to https://github.com/Geekble-Maker/Geekble_Note2Freq.git
Success!
```

**Pull request merged**
